cmd/cluster/core/create.go

```diff
@@ -440,7 +432,11 @@ func CreateCluster(ctx context.Context, opts *CreateOptions, platformSpecificApp
 func defaultNetworkType(ctx context.Context, opts *CreateOptions, releaseProvider releaseinfo.Provider, readFile func(string) ([]byte, error)) error {
 	if opts.NetworkType != "" {
 		return nil
+	} else if opts.ReleaseImage == "" {
```

Comment:
I think this change is causing the unit test failure

Reply:
ah, yep, fixed